nucliadb_utils/nucliadb_utils/settings.py

```diff
@@ -68,11 +68,25 @@ class StorageSettings(BaseSettings):
         FileBackendConfig.NOT_SET, description="File backend storage type"
     )
 
-    gcs_base64_creds: Optional[str] = None
-    gcs_bucket: Optional[str] = None
-    gcs_location: Optional[str] = None
-    gcs_project: Optional[str] = None
-    gcs_bucket_labels: Dict[str, str] = {}
+    gcs_base64_creds: Optional[str] = Field(
+        None, description="GCS JSON credentials encoded in Base64"
```

Comment:
@nuclia/nuclia-db which kind of authentication are we requiring for nucliadb when GCS? I would like to add a link to the google documentation but I am not sure about this

Reply:
I think this credentials are from a service account with permissions to write on bucket storage